<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>657</width>
         <height>518</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_image">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background-color: rgb(149, 149, 149);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_1">
             <property name="text">
              <string>曝光检测</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_2">
             <property name="text">
              <string>去雾</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_3">
             <property name="text">
              <string>去暗角</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_4">
             <property name="text">
              <string>运动去模糊</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_5">
             <property name="text">
              <string>磨皮美颜</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_6">
             <property name="text">
              <string>去红眼</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_7">
             <property name="text">
              <string>直方图均衡化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0">
           <item>
            <widget class="QPushButton" name="pB_check_8">
             <property name="text">
              <string>ACE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_8">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="pB_check_9">
             <property name="text">
              <string>霓虹滤镜</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_result_9">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pB_check">
             <property name="text">
              <string>检测所有</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB_tuning">
             <property name="text">
              <string>美化所有</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
